README.md

```diff
@@ -1,2 +1,33 @@
 # CyUSBSerial_SDK
-Installer repository for Cypress USB serial controller drivers
+Installer repository for Cypress USB serial controller drivers.
+
+Specifically creates a CMake debian installer, but there's no reason this project can't be used on other platforms with some modification.
+
+To make an installable linux package
+
+```shell
+mkdir build
+cd build
+cmake .. -DCMAKE_BUILD_TYPE=Release
+make -j$(nproc) package
+```
+
+This should create a `*.deb` file in the folder. To install, just run
+`sudo apt install ./*.deb`
```

Comment:
Do we want to also add `sudo dpkg -i ./*.deb`?

Reply:
If you think that is necessary. I was going to add an uninstall instruction as well, just to make it clear it has that functionality properly implemented.

Reply:
Not necessary.